sql/parser/parse_test.go

```diff
@@ -30,6 +30,10 @@ func TestParse(t *testing.T) {
 	testData := []struct {
 		sql string
 	}{
+		// TODO(tschottdorf): shouldn't these below error?
+		{``},
+		{`VALUES ("")`},
```

Comment:
`VALUES` is actually considered a type of "select statement".  I don't know if this is ANSI SQL, but postgres allows it:

```
pmattis=# VALUES (1, 2);
 column1 | column2
---------+---------
       1 |       2
(1 row)
```